com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2021.1-to-2021.2.md

```diff
@@ -80,3 +80,5 @@ If you used transparent materials (**Surface Type** set to **Transparent**) with
 From 2021.2, HDRP includes an updated `RendererList` API in the `UnityEngine.Rendering.RendererUtils` namespace. This API has lower overhead than the previous version of the `RendererList` API when it submits the RendererList for drawing. You can use this new version to query if the list of visible objects is empty.
```

Comment:
I think we can clarify "lower overhead" here, based on your description: 

This API performs fewer operations than the previous version of the `RendererList` API when it submits the RendererList for drawing.

Reply:
Sounds good, thanks.